recipes/pyexcel-webio/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "pyexcel-webio" %}
+{% set version = "0.1.2" %}
+{% set sha256 = "58f9a310dc3861360bc02e2246d5501dea48c342a77575dc09030d12f1406956" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pyexcel >=0.5.1
```

Comment:
Is `pyexcel` an actual `build` requirement?

Reply:
It is not necessary any extra package, you are correct. I was basing my recipe on the conda skeleton and looking at setup.py I noticed that is not necessary any package for the build process.